great-idea/index.html

```diff
@@ -17,7 +17,87 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+           
+    <div class="navigation">  
+      <nav>
+        <a href="#">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+           <img class="logo" src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/logo.png" alt="Great Idea! Company logo.">
+   </div>
+   
+   <header>
+     <div class="header">
+    <div class="demand">
+     <h1> Innovation<br>
+     On<br>
+     Demand<br>
+     </h1>
+       </div>
       
     
+       
+     
+     <img class="top-image"src="https://raw.githubusercontent.com/LambdaSchool/User-Interface/master/great-idea-website/img/header-img.png" alt="Image of a code snippet.">
+     </div>
+    
+     
+    <div class="click">
+     <button> Get Started </button>
+    </div>
+     
+    </header>
+    
+   <section class="top-content">    
+     <div class="top">
+     <h3> Features </h3>
+     Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+     </div>
+   
+     <div class="top">
+     <h3> About </h3>
+     <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+     </div>  
```

Comment:
I see you used a p tag here, but no where else. You want to wrap all of that floating text in a P tags. You want all of the content to be marked up in tags.